src/ocm/services/apis/ClustersAPI.ts

```diff
@@ -178,7 +178,7 @@ const ClustersAPI = {
 
   listByOpenshiftId(openshiftId: Cluster['openshiftClusterId']) {
     return client.get<Cluster[]>(
-      `${ClustersAPI.makeBaseURI()}?openshift_cluster_id=${openshiftId}`,
+      `${ClustersAPI.makeBaseURI()}?openshift_cluster_id=${openshiftId || ''}`,
```

Comment:
In this case, if `openshiftId` is undefined then there is no point in calling this API

Reply:
Agree, it's just that with the current type definition it needs the fallback for TS.